frontend/packages/dev-console/integration-tests/features/web-terminal/web-teminal-basic.feature

```diff
@@ -3,29 +3,28 @@ Feature: Web Terminal
               As a basic user, I should be able to use web terminal
 
         Background:
-            Given user has logged in as basic user
-              And  user has created or selected namespace "aut-terminal-basic"
-
+            Given user with basic rights has installed Web Terminal operator
+              And user has installed webTerminal in namespace "aut-terminal-basic"
+              And user has logged in as basic user
+              And user has created or selected namespace "aut-terminal-basic"
 
         @regression @to-do
         Scenario: Open existing Web Terminal instance: WT-01-TC01
-            Given user has installed Web Terminal operator
              When user clicks on the Web Terminal icon on the Masthead
              Then user will see the terminal window
               And user close current Web Terminal session
 
-
         @regression @to-do
         Scenario: Web Terminal in new tab: WT-01-TC02
             Given user can see terminal icon on masthead
              When user clicks on the Web Terminal icon on the Masthead
               And user clicks on Open Terminal in new tab button on the terminal window
              Then user will see the terminal window opened in new tab
 
-
         @regression @to-do
-        Scenario: Web Terminal is stopped by inactivity: WT-01-TC03
+        Scenario: user does nothing with displayed terminal window 1 minutes: WT-01-TC03
```

Comment:
Hey, the new scenario title says what the user does, not what is expected. I would recommend to keep the old title for now or is there any reason to change this title?

```suggestion
        Scenario: Web Terminal is stopped by inactivity: WT-01-TC03
```